shared/control/parallel_dd.control

```diff
@@ -12,13 +12,8 @@ the files system.
 """
 
 # YOU NEED TO SPECIFY A FILESYSTEM
-path = glob.glob("/dev/[hsv]d?")
-path.sort()
-fs = path[-1]
-platform = Popen(["uname", "-m"], stdout=PIPE).communicate()[0]
-if "ppc" in platform:
-    if not Popen("lsscsi", stdout=PIPE).communicate()[0]:
-        fs = path[0]
+path = glob.glob("/dev/disk/by-id/*TARGET_DISK*")
```

Comment:
```
>>> from subprocess import Popen, PIPE
>>> 
>>> path = glob.glob("/dev/disk/by-id/*qazwsx*")
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'glob' is not defined